realm/realm-library/src/main/java/io/realm/OrderedRealmCollectionImpl.java

```diff
@@ -60,8 +59,8 @@ Collection getCollection() {
         return collectionOperator.getCollection();
     }
 
-    boolean forValue() {
-        return collectionOperator.forValue();
+    boolean forModelClasses() {
```

Comment:
this was renamed to mean its opposite but the values it provides is still `return true`, is that ok?

Reply:
I changed it because having a method called `forValue()` return `false` for an `int` sounded really wrong 😄

Reply:
I agree, that does sound wrong!